eZ/Publish/API/Repository/URLAliasService.php

```diff
@@ -118,11 +137,18 @@ public function lookup($url, $languageCode = null);
      * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if no url alias exist for the given language
      *
      * @param \eZ\Publish\API\Repository\Values\Content\Location $location
-     * @param string $languageCode
+     * @param string|null $languageCode
+     * @param bool|null $showAllTranslations
+     * @param string[]|null $prioritizedLanguages
```

Comment:
@adamwojs if you're touching PHPDocs in the scope, then you should drop all attributes which don't provide any extra information with respect to the method prototype (use [inspections profile](https://github.com/ezsystems/ezplatform-code-style/pull/1/files) to see what really is needed (e.g. `array` is considered non-strict and needs to be documented)